app/src/ereferrals/java/org/eyeseetea/malariacare/domain/boundary/IElementController.java

```diff
@@ -0,0 +1,9 @@
+package org.eyeseetea.malariacare.domain.boundary;
```

Comment:
As explained before, I think we need to generalize much more this interface to any post-send action, and not only for Element.